docs/source/contact.html.haml

```diff
@@ -67,6 +79,7 @@ subtitle: Turn any <code>&lt;input&gt;</code> into an address autocomplete
     var formState = getUrlParameter('form');
 
     if (formState === 'success'){
+      debugger;
```

Comment:
is this wanted?

Reply:
Oh shit good call

Reply:
There is one or two console debug as well 😬

Reply:
Anyway they changed the forms again so I'm gonna push an edit